setup.py

```diff
@@ -126,6 +126,7 @@ def initialize_options(self) -> None:
 
 itest_dependencies = [
     'jsonschema',  # needed for directory_validator.py
+    'pytz',
```

Comment:
This file, one of the 3 largest in the project, increased in size to 366 lines.  The total size of those files is now 3408 lines (target: 1135).  Is this file complex enough to refactor?